packages/@blockprotocol/graph/src/graph-block-handler.ts

```diff
@@ -1,6 +1,13 @@
 import { ServiceHandler } from "@blockprotocol/core";
 
-import serviceJsonDefinition from "./graph-service.json";
+/**
+ * There's an issue when importing useGraphEmbedderService from @blockprotocol/graph/react in hashintel/hash:
+ * NextJS's output file tracing does not include graph-service.json, and yet an import statement for it is preserved.
+ * This leads to a 'module cannot be found error'. For now, commenting out the import of the JSON from this file.
+ * @todo restore this when module resolution issue resolved
+ * @see https://app.asana.com/0/1202542409311090/1202614421149286/f
+ */
+// import graphServiceJson from "./graph-service.json" assert { type: "json" };
```

Comment:
I guess we can use `readFileSync` instead, which should be picked up by [verce/nft](https://github.com/vercel/nft). A call to this function can be placed in `graph-service-json.ts`. Not a blocker, just something that can save us another release.

Reply:
Good idea, thank you - I will try this in a follow up as @teenoh needs this merged quicker for a PR based on it. 

tracked [here](https://app.asana.com/0/1202538466812818/1202614421149286/f)